tests/test_image_to_text_example.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+import re
+import subprocess
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import pytest
+
+from .test_examples import TIME_PERF_FACTOR
+
+
+if os.environ.get("GAUDI2_CI", "0") == "1":
+    # Gaudi2 CI baselines
+    MODELS_TO_TEST = {
+        "bf16": [
+            ("llava-hf/llava-1.5-7b-hf", 1, False, 264.1947269439697),
+        ],
+    }
+
+def _test_image_to_text(
+    model_name: str,
+    baseline: float,
+    token: str,
+    batch_size: int = 1,
+    reuse_cache: bool = False,
+    deepspeed: bool = False,
+    world_size: int = 8,
+    torch_compile: bool = False,
+    fp8: bool = False,
+):
+    command = ["python3"]
+    path_to_example_dir = Path(__file__).resolve().parent.parent / "examples"
+    env_variables = os.environ.copy()
+
+    command += [
+        f"{path_to_example_dir / 'image-to-text' / 'run_pipeline.py'}",
+        f"--model_name_or_path {model_name}",
+        f"--batch_size {batch_size}",
+        "--use_kv_cache",
+        "--max_new_tokens 20",
```

Comment:
have you ran the test with
 GAUDI2_CI=1 RUN_SLOW=true  python -m pytest tests/test_image_to_text_example.py -v -s
if so, you will see run_pipeline.py: error: unrecognized arguments: --use_kv_cache --output_dir /tmp/tmpsp9f6li_ --token None
you should include whatever arguments as  python3 run_pipeline.py \
        --model_name_or_path "llava-hf/llava-1.5-7b-hf" \
        --image_path "https://llava-vl.github.io/static/images/view.jpg" \
        --prompt "<image>\nUSER: What's the content of the image?\nASSISTANT:" \
        --max_new_tokens 20 \
        --use_hpu_graphs \
        --bf16

Reply:
updated